consensus/spos/bls/blsSubroundsFactory.go

```diff
@@ -73,6 +75,11 @@ func (fct *factory) SetAppStatusHandler(ash core.AppStatusHandler) error {
 	return nil
 }
 
+// SetIndexer method set indexer
```

Comment:
change the comment to something like _SetIndexer will update the value of the factory's indexer_ . Also do the same thing at line 68

Reply:
Done.